python/mujoco/rollout_test.py

```diff
@@ -461,7 +461,7 @@ def test_threading(self):
     model = mujoco.MjModel.from_xml_string(TEST_XML)
     nstate = mujoco.mj_stateSize(model, mujoco.mjtState.mjSTATE_FULLPHYSICS)
     num_workers = 32
-    nroll = 10000
+    nroll = 100
```

Comment:
I reduced the number of rollouts for all threading tests. They were taking a noticeable amount of time to run.